packages/dev/babel-preset/index.js

```diff
@@ -15,6 +16,12 @@ module.exports = () => ({
     require('@babel/plugin-proposal-class-properties'),
     require('@babel/plugin-proposal-nullish-coalescing-operator'),
     require('@babel/plugin-proposal-optional-chaining'),
+    [
+      require('@babel/plugin-transform-modules-commonjs'),
+      {
+        lazy: () => process.env.PARCEL_BUILD_ENV !== 'test',
```

Comment:
Not entirely sure why, but lazy mode seems to hang the tests completely. Maybe something with mocha?